actors/common/src/main/java/org/sunbird/models/Category.java

```diff
@@ -0,0 +1,15 @@
+package org.sunbird.models;
+
+import org.sunbird.common.models.util.JsonKey;
+
+public enum  Category {
+
+    MIGRATION_USER(JsonKey.MIGRATION_USER_OBJECT);
```

Comment:
why enum value is MIGRATION_USER_OBJECT.
i am thinking to use "USER_EXTERNAL_ID_VALIDATION" as enum name and value as well.